README.md

```diff
@@ -167,6 +171,16 @@ the  `-s` option with `types productTypes` as below:
 -s types productTypes
 ```
 
+#### Running Product Sync with custom fetchLimit and customQuery
+
+The application can sync products with the custom values provided as an argument. For example, to run `products` sync with custom values, 
```

Comment:
I would document the same terms, already provided in CTP docs so users could understand the concept better.

#### Running ProductSync with custom product query parameters

You might pass your customized product fetch limit and a product predicate to filter product resources to sync in the JSON format. 

For instance:

```bash
-s products -productQueryParameters {"limit": 100, "where": "masterData(published=true) AND masterData(staged(masterVariant(attributes(name="attribute-name" and value="attribute-value")))))"}
```

Predicates provide a way for complex filter expressions when querying resources. Refer [commercetools docs](https://docs.commercetools.com/api/predicates/query) for more details. 

Note: The value of the `productQueryParameters` argument should be in JSON format and as shown in the above example, please use escape character \ for the nested double quote values.

Reply:
Could you give a valid example (with escape thingy)?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/243/commits/1f200748b2537c38754f63aa71e39b4f9bd35c37

Reply:
Instead of giving 2 similar examples, could you please give one example with escaped quotes?
Also, it comes to my mind now, are we supporting only limit or only a query, so we are good and have defaults when it's not set?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/243/commits/3f76c4af0233b0d433c9dafaa99048daf1ae01c7.   https://github.com/commercetools/commercetools-project-sync/pull/243/commits/55edd703d24e7a5799ddf9b27e76cd819779a4d7

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/243/commits/41d439eecf8707b907a89dd8c2deef9831919b3d